packages/build/src/npm-packages/bump.spec.ts

```diff
@@ -35,9 +46,112 @@ describe('npm-packages bump', function () {
       expect(fsWriteFile).not.called;
       consoleWarnSpy.restore();
     });
+
+    it('bumps only mongosh packages', async function () {
+      getPackagesInTopologicalOrder.resolves([
+        { name: 'mongosh', location: 'packages/mongosh' },
+        { name: '@mongosh/autocomplete', location: 'packages/autocomplete' },
+      ]);
+
+      const rootProjectJson = `${PROJECT_ROOT}/package.json`;
+      const mockPackageJson = [
+        [
+          rootProjectJson,
+          {
+            name: 'mongosh',
+            devDependencies: {
+              mongosh: '0.1.2',
+              '@mongosh/cli-repl': '0.1.2',
+              '@mongosh/autocomplete': '1.2.3',
+            },
+          },
+        ],
+        [
+          'packages/mongosh/package.json',
+          {
+            name: 'mongosh',
+            version: '0.1.2',
+            devDependencies: {
+              '@mongosh/cli-repl': '9.9.9',
+              '@mongosh/autocomplete': '1.2.3',
+            },
+          },
+        ],
+        [
+          'packages/autocomplete/package.json',
+          {
+            name: '@mongosh/autocomplete',
+            version: '1.2.3',
+            devDependencies: {
+              '@mongosh/cli-repl': '0.1.2',
+              '@mongosh/config': '3.3.3',
+            },
+          },
+        ],
+      ];
+
+      fsReadFile.throws('Unknown file');
+      for (const [file, json] of mockPackageJson) {
+        fsReadFile.withArgs(file, 'utf8').resolves(JSON.stringify(json));
+      }
+
+      const updateShellApiMongoshVersion = sinon.stub();
+      await bumpMongoshReleasePackages(
+        '9.9.9',
+        getPackagesInTopologicalOrder,
+        updateShellApiMongoshVersion
+      );
+      expect(fsWriteFile).callCount(3);
+
+      expect(
+        JSON.parse(
+          fsWriteFile.withArgs(rootProjectJson).firstCall.args[1] as string
+        )
+      ).deep.equals({
+        name: 'mongosh',
+        devDependencies: {
+          mongosh: '9.9.9',
+          '@mongosh/cli-repl': '9.9.9',
+          '@mongosh/autocomplete': '1.2.3',
+        },
+      });
+
+      expect(
+        JSON.parse(
+          fsWriteFile.withArgs('packages/mongosh/package.json').firstCall
+            .args[1] as string
+        )
+      ).deep.equals({
+        name: 'mongosh',
+        version: '9.9.9',
+        devDependencies: {
+          '@mongosh/cli-repl': '9.9.9',
+          '@mongosh/autocomplete': '1.2.3',
+        },
+      });
+
+      expect(
+        JSON.parse(
+          fsWriteFile.withArgs('packages/autocomplete/package.json').firstCall
+            .args[1] as string
+        )
+      ).deep.equals({
+        name: '@mongosh/autocomplete',
+        version: '1.2.3',
```

Comment:
We do want to bump these packages, just not with the same version as mongosh, right?

Otherwise the published `@mongosh/cli-repl` package.json would specify a version that's not actually the same code as the one in the compiled executable?

Reply:
yeah, this happens at the previous step with `bumpAuxiliaryPackages`

Reply:
it'd be good to have a test mixing the two though yes